.github/workflows/tests.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+name: Tests
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: [8.1, 8.2]
+        symfony-version: ["^5.2", "^6.3"]
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          tools: composer
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: composer install
```

Comment:
You could either run `composer install` and after that `composer require symfony/framework-bundle=${{ matrix.symfony-version }}` to specifiy the version or use a different approach and run the tests against the newest version of all dependencies and against the lowest version of all dependencies (which afaik the recommended approach for libraries).

This can be done by running just `composer update` without any options to get the latest versions of the dependencies according to the composer.json and with another matrix job running `composer update --prefer-lowest --prefer-stable` to use the lowest allowed versions.